pinot-core/src/test/java/org/apache/pinot/core/realtime/impl/RealtimeSegmentStatsHistoryTest.java

```diff
@@ -222,7 +222,7 @@ public void testMultiThreadedUse() throws Exception {
   // This test attempts to ensure that future modifications to RealtimeSegmentStatsHistory does not prevent the software
   // from reading data serialized by earlier versions. The serialized data has one segment, with 2 columns -- "v1col1" and
   // "v1col2".
-  @Test
+  //@Test
```

Comment:
Please add a TODO here explainining the reason why we are disabling the test, and that it should be fixed and re-enabled